### DISCUSSION TITLE
Add musl release builds

### DISCUSSION THREAD

**Discussion by RibalGZ** [godotengine/godot-proposals#7438] [General Discussions]
### Describe the problem or limitation you are having in your project

I have to compile the engine on every dev release.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be nice to have musl based release builds available in the downloads section.

**Comment by Calinou** [comment]
As I understand it, it's not technically possible to link an app that uses OpenGL or Vulkan against musl, due to graphics drivers not being relinkable. You don't have the source code of the NVIDIA driver after all :slightly_smiling_face: 

Headless-only binaries can already be linked against musl, but there are no plans to provide official binaries this way. That said, nothing prevents musl-focused distributions such as Alpine Linux from providing packages (along with container builds, which is where musl is most relevant).
